src/components/card/Card.tsx

```diff
@@ -1,34 +1,32 @@
-import { CARD_INFO, SPACE, SPACE_REPEATING } from '../../constants/constants';
-import React from 'react';
-import { TCard } from '../../types/types';
+import React, { useState } from 'react';
+import Skeleton from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+import { FoundItem, Item } from '../../types/types';
 import './Card.css';
+import { getItemById } from '../../api/getItemById';
+import { DEFAULT_IMG } from '../../constants/constants';
+
+const Card = (props: { card: FoundItem | Item; openPortal: (item: Item) => void }) => {
```

Comment:
Better to use interfaces for props and here you can destructure props as well. Like const Card = ( {card}: CardProps) => {}